packages/css/src/components/Modal/modal.css

```diff
@@ -0,0 +1,52 @@
+.modal {
+    position: absolute;
+    max-width: 660px;
+    width: 100%;
+    border-radius: var(--modal-border-radius);
+    background: var(--modal-background-color);
+    box-shadow: var(--modal-shadow);
+}
+
+.modal-header {
+    padding: var(--padding-3) var(--padding-5);
+    border-bottom: 1px solid var(--modal-border-color);
+    color: var(--modal-header-color);
+    font: var(--modal-header-font);
+}
+
+.modal-body {
+    border-bottom: 1px solid var(--modal-border-color);
+    color: var(--modal-body-color);
+    font: var(--modal-body-font);
+}
+
+.modal-footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    padding: var(--padding-3) var(--padding-5);
+    gap: var(--margin-3);
+}
+
+.modal-button-close {
+    position: absolute;
+    top: calc(var(--padding-3) + 4px);
+    right: var(--padding-5);
+    font-size: 24px;
```

Comment:
Why use a fixed font size here?

Reply:
That's to make the icon match the design. We can decide later how to handle that, once we have the icons implemented.